# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../eg/gcc33.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
gcc3.3 don't support multiline strings
	Alejandro

--- eg-1.00.orig/eg.c	2000-07-04 09:18:49.000000000 -0400
+++ eg-1.00/eg.c	2004-02-02 17:18:35.000000000 -0300
@@ -174,24 +174,23 @@
     uname( &utsn );
 
     printf( "Expert Guide Version " EG_VERSION "\n\n"
-            "\
-     Expert Guide - A Text Mode Norton Guide Reader
-     Copyright (C) 1997,1998,1999,2000 David A Pearson
-   
-     This program is free software; you can redistribute it and/or modify
-     it under the terms of the GNU General Public License as published by
-     the Free Software Foundation; either version 2 of the license, or 
-     (at your option) any later version.
-     
-     This program is distributed in the hope that it will be useful,
-     but WITHOUT ANY WARRANTY; without even the implied warranty of
-     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-     GNU General Public License for more details.
-     
-     You should have received a copy of the GNU General Public License
-     along with this program; if not, write to the Free Software
-     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
-\n" );
+"     Expert Guide - A Text Mode Norton Guide Reader\n"
+"     Copyright (C) 1997,1998,1999,2000 David A Pearson\n"
+"\n"
+"     This program is free software; you can redistribute it and/or modify\n"
+"     it under the terms of the GNU General Public License as published by\n"
+"     the Free Software Foundation; either version 2 of the license, or\n"
+"     (at your option) any later version.\n"
+"\n"     
+"     This program is distributed in the hope that it will be useful,\n"
+"     but WITHOUT ANY WARRANTY; without even the implied warranty of\n"
+"     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the\n"
+"     GNU General Public License for more details.\n"
+"\n"     
+"     You should have received a copy of the GNU General Public License\n"
+"     along with this program; if not, write to the Free Software\n"
+"     Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.\n"
+"\n" );
     
     printf( "System details:\n"
             "\tSystem.......: %s %s\n"
